x-pack/plugins/security_solution/public/common/components/url_state/use_url_state.tsx

```diff
@@ -221,7 +221,7 @@ export const useUrlStateHooks = ({
         }
       });
     } else if (pathName !== prevProps.pathName) {
-      handleInitialize(type, pageName === SecurityPageName.detections);
+      handleInitialize(type, pageName === SecurityPageName.alerts);
```

Comment:
I also think we need to add rule details

Reply:
fixed in https://github.com/elastic/kibana/pull/103659